add number of comments

tianlangwu opened the pull request:
This RP is for [issue-57](https://github.com/trybick/issue-collab/issues/57). The number of comments shows on the bottom left of each issue.
![SS02](https://user-images.githubusercontent.com/60892464/96351303-73d70480-1088-11eb-97ec-0b957952ae03.jpg)

When I am run the command `npm run lint:fix` to check my formatting, it automatically formats something in the other files.

Let me know if I did anything incorrectly, thanks for the consideration.

trybick commented on src/components/search/SearchResultsContainer.jsx:
```diff
@@ -21,7 +21,7 @@ const SearchResultsContainer = ({ currentPage, onPageChange, results }) => {
     results.items[0] &&
     results.items.map(item => {
-      const htmlUrl = item.html_url.split('/');
+      const { htmlUrl, comments } = [item.html_url.split('/'), item.comments];
```
Would it be possible to do this on a separate line than the htmlUrl - 
`const { comments } = item;`

tianlangwu replied:
I used to do it this way, 
![ss03](https://user-images.githubusercontent.com/60892464/96352155-28742480-108f-11eb-9675-66f02f1c633b.jpg)

but when I run the formating command, it said 
![ss04](https://user-images.githubusercontent.com/60892464/96352176-4772b680-108f-11eb-83d5-716c4b7ddc69.jpg)

trybick replied:
Oh gotcha. Please skip the lint:fix command for now

trybick commented on src/components/search/SearchResult.jsx:
```diff
@@ -38,7 +39,8 @@ export const SearchResult = ({
         <div className="metadata">
-          <div>{`${userName}/${repoName}`}</div>
+          <div>{`${userName}/${repoName}`} </div>
+          <div className="comments">{`Total comments ${comments}`}</div>
```
Could we do - 

`Comments: ${comments}`

tianlangwu replied:
Sure, thanks for the advice.

trybick reviewed (commented):
It looks really good 👍  However I am getting this error when I click the 'Get Issues' in the deploy preview, can you take a look? This might be fixed by resolving the comment about this line.

<img width="562" alt="Screen Shot 2020-10-17 at 3 28 46 PM" src="https://user-images.githubusercontent.com/39889198/96351990-bea74b00-108d-11eb-830f-b0664479c91e.png">

trybick commented:
Also sorry, there are some merge conflicts which were introduced by this PR  (if it helps to look at it 
 https://github.com/trybick/issue-collab/pull/60)

tianlangwu committed:
add number of comments

tianlangwu commented:
> Also sorry, there are some merge conflicts which were introduced by this PR (if it helps to look at it
> #60)

I guess it is because I ran the `npm run lint:fix`, and it changed some format in some files. 
I have fixed it, can you please check is my PR  fine now? I have fixed the problem on the deploy preview too.

trybick commented:
Awesome, thanks @tianlangwu !
